rabbitmq-operator-model/src/main/java/com/indeed/operators/rabbitmq/model/crd/rabbitmq/OperatorPolicyDefinitionSpec.java

```diff
@@ -0,0 +1,64 @@
+package com.indeed.operators.rabbitmq.model.crd.rabbitmq;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.indeed.rabbitmq.admin.pojo.OperatorPolicyDefinition;
+
+public class OperatorPolicyDefinitionSpec {
+
+    private final Long expires;
```

Comment:
Maybe javadoc/link to documentation for the meaning of these values?

Reply:
i'll knock out javadoc for everything in a followup issue

Reply:
#39